timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -28,11 +36,13 @@ types:
           requestId: uuid
           requestorId: uuid
           numTransactions: integer
+          lastKnownVersion: LastKnownVersion
       ConjureStartTransactionsResponse:
         fields:
           immutableTimestamp: LockImmutableTimestampResponse
           timestamps: PartitionedTimestamps
           lease: Lease
+          lockWatchUpdate: LockWatchStateUpdate
```

Comment:
This is not a breaking change, you are adding an optional field. It's just that this will be hidden from conjure backcompat the way it is written

Reply:
Yeah, the response is fine, but the request is the break

Reply:
the request parameter is _also_ optional, and so won't be a break either.